packages/request-client.js/src/api/request.ts

```diff
@@ -171,9 +185,13 @@ export default class Request {
     // refresh the local request data
     const requestData = await this.refresh();
 
-    cancelResult.on('confirmed', async () => {
-      requestData.emit('confirmed', await this.refresh());
-    });
+    cancelResult
+      .on('confirmed', async () => {
+        requestData.emit('confirmed', await this.refresh());
+      })
+      .on('error', error => {
+        this.emitter.emit('error', error);
```

Comment:
Same as above comment (and same comment for all these kind of line), why not setting `confirmationErrorOccurred` ?

Reply:
same same